src/Cli/dotnet/Parser.cs

```diff
@@ -167,6 +167,7 @@ public static bool TokenPerLine(string tokenToReplace, out IReadOnlyList<string>
         {
             EnableDefaultExceptionHandler = false,
             EnablePosixBundling = false,
+            Directives = { new DiagramDirective(), new SuggestDirective(), new EnvironmentVariablesDirective() },
```

Comment:
@baronfel This change won't work in `main` because the structure changed for how directives are handled in recent S.CL versions. Directives exist on the `CliRootCommand` but not on `CliConfiguration`.

Reply:
Turned out to be a simple swap around!